### DISCUSSION TITLE
GitHub Education Approved but not active

### DISCUSSION THREAD

**Discussion by Adelh0lzener** [community/community#140429] [GitHub Education] [answered]
Hello,

I recently applied for GitHub Education and my application was approved allready. I see the following message:

`Congrats! Your request for @Adelh0lzener was approved and your benefits will be available soon.`

My benefits are still not active, and I am unable to access them. Could you please look into this and help activate my GitHub Education benefits?

Thank you in advance!

Best regards
@Adelh0lzener

**Comment by henry-AY** [comment]
Hey Adelh0lzener,

Educational benefits generally arrive within 72 hours of being verified. However, during busy periods such as the start of a new semester or academic year, it can take a little longer :)

Best,
@Hy8012

**Comment by davevad93** [comment, marked as answer]
Hi @Adelh0lzener ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Adelh0lzener was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
